pkg/arvo/sys/vane/clay.hoon

```diff
@@ -1307,21 +1333,18 @@
     ?~  ref
       (run-if-future rove.wov |=(@da (bait hen +<)))
     |-  ^+  +>+.$
-    =/  =rave  (rove-to-rave rove.wov)
-    =.  rave
-      ?.  ?=([%sing %v *] rave)  rave
-      [%many %| [%ud let.dom] case.mood.rave path.mood.rave]
```

Comment:
This transmutation is load-bearing and matches with the TODO in +apply-foreign-update.  Worth paring on this, but the gist is that `%sing %v` ends up producing a response `+dome:clay` (similar to the +dome defined in clay.hoon but without some of the caches).  But if that request goes over the network, it actually gets a +nako over the network instead of a +dome.

You can think of a +nako as "+dome without even the ankh cache because that's too big, plus the commits referred to in the dome", and when you receive one you need to scan the commits for unknown blobs and request those (the +foreign-update state machine, starting from +apex).

I suspect we can do this without a transmute-to-%many step by simply recognizing at the edges that we're about to send a %v request over the wire (so convert +dome to +nako) or being received over the wire (so convert +dome to +nako and fetch blobs).  Would be much simpler.